internal/pkg/cli/run_local.go

```diff
@@ -354,6 +371,47 @@ func (o *runLocalOpts) Execute() error {
 	}
 }
 
+func (o *runLocalOpts) getContainerIDs(ctx context.Context) (map[string]string, error) {
```

Comment:
nit nit nit: since we are getting containers specifically with exec enabled......
```suggestion
func (o *runLocalOpts) getExecEnabledContainerIDs(ctx context.Context) (map[string]string, error) {
```

Reply:
i did something slightly different as part of the other refactor, lmk what you think of the new name! (`getSSMTarget()`)